### DISCUSSION TITLE
Chia harvester error

### DISCUSSION THREAD

**Discussion by byzilker** [Chia-Network/chia-blockchain#8604] [Support]
i've this errors in my log data please help.

harvester chia.harvester.harvester: ERROR    Error using prover object Invalid file E:\plot-k32-2021-09-11-10-58-a171535409e020cdd1dbf83ca95d9e0c8fba530ba92ee767040d420e7ac81ea4.plot

harvester chia.harvester.harvester: ERROR    Error using prover object Invalid file H:\plot-k32-2021-08-25-07-11-6f96fa730e9d77b20fdc0e2d6dc843ba323042afaedd194173b52c62283ed155.plot

all of my plots have error, but gui is farming..

**Comment by loppefaaret** [comment]
the harvester is having problems reading the files - try restarting the app - try and ensure that the drives are connected and mounted correctly, that you got permissions to access them, if all else fails, try `chia plots check` from the CLI:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference

**Comment by BrandtH22** [comment]
@byzilker , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
